ha-addon/mqtt_discovery/qheat.json

```
@@ -0,0 +1,50 @@
+{
+    "total_energy_consumption_kwh": {
+        "component": "sensor",
+        "discovery_payload": {
+            "device": {
+                "identifiers": [
+                    "wmbusmeters_{id}"
+                ],
+                "manufacturer": "Qundis",
+                "model": "{driver}",
+                "name": "{name}",
+                "sw_version": "{id}"
+            },
+            "enabled_by_default": true,
+            "json_attributes_topic": "wmbusmeters/{name}",
+            "device_class": "energy",
```

```suggestion
            "device_class": "gas",
```

While technically incorrect I had to change this in order to being able to include my qHeat meter into the Energy dashboard as main source for Heat instead of the Energy section that usually tracks electrical energy.

---

Yeah I was aware of that HA shortcoming, but decided that this is a workaround that anyone could do on their own by changing the default file if needed.
I am not using that for now and instead track heating as a separate device because it looks wrong to me to have my heating listed as gas.

Please support https://community.home-assistant.io/t/energy-dashboard-change-gas-into-heating/391647 to add a separate heating category to the energy dashboard.